thorlcr/master/thmastermain.cpp

```diff
@@ -1128,32 +1114,28 @@ int main( int argc, const char *argv[]  )
  * NB: This isn't a real StoragePlane, and it will not be accessible by any other component.
  *
  */
-                StringBuffer uniqueGrpName;
-                queryNamedGroupStore().addUnique(&queryProcessGroup(), uniqueGrpName);
-                // change default plane
-                getComponentConfigSP()->setProp("@dataPlane", uniqueGrpName);
-                PROGLOG("Persistent Thor group created with group name: %s", uniqueGrpName.str());
-            }
+            StringBuffer uniqueGrpName;
+            queryNamedGroupStore().addUnique(&queryProcessGroup(), uniqueGrpName);
+            // change default plane
+            getComponentConfigSP()->setProp("@dataPlane", uniqueGrpName);
+            PROGLOG("Persistent Thor group created with group name: %s", uniqueGrpName.str());
+        }
 #endif
-            LOG(MCauditInfo, ",Progress,Thor,Startup,%s,%s,%s,%s",nodeGroup.str(),thorname,queueName.str(),logUrl.str());
-            auditStartLogged = true;
+        LOG(MCauditInfo, ",Progress,Thor,Startup,%s,%s,%s,%s",nodeGroup.str(),thorname,queueName.str(),logUrl.str());
+        auditStartLogged = true;
 
-            writeSentinelFile(sentinelFile);
+        writeSentinelFile(sentinelFile);
 
 #ifndef _CONTAINERIZED
-            unsigned pinterval = globals->getPropInt("@system_monitor_interval",1000*60);
-            if (pinterval)
-                startPerformanceMonitor(pinterval, PerfMonStandard, nullptr);
+        unsigned pinterval = globals->getPropInt("@system_monitor_interval",1000*60);
+        if (pinterval)
+            startPerformanceMonitor(pinterval, PerfMonStandard, nullptr);
 #endif
-            configurePreferredPlanes();
+        configurePreferredPlanes();
 
-            // NB: workunit/graphName only set in one-shot mode (if isCloud())
-            thorMain(logHandler, workunit, graphName);
-            LOG(MCauditInfo, ",Progress,Thor,Terminate,%s,%s,%s",thorname,nodeGroup.str(),queueName.str());
-        }
-        else
-            PROGLOG("Registration aborted");
-        registry.clear();
+        // NB: workunit/graphName only set in one-shot mode (if isCloud())
+        thorMain(logHandler, workunit, graphName);
+        LOG(MCauditInfo, ",Progress,Thor,Terminate,%s,%s,%s",thorname,nodeGroup.str(),queueName.str());
         LOG(MCdebugProgress, thorJob, "ThorMaster terminated OK");
```

Comment:
Does it make any difference that this is now before the destruction of registry?